batch/batch/front_end/front_end.py

```diff
@@ -500,9 +500,10 @@
 ''',
                                           jobs_args)
                 except pymysql.err.IntegrityError as err:
-                    if err.args[1] == 1022:
+                    if err.args[1] in (1022, 1062, 1586, 1859, 3026):
                         log.info(f'bunch containing job {(batch_id, jobs_args[0][1])} already inserted ({err})')
                         return web.Response()
```

Comment:
This must be wrong.  The return value to the call to insert is ignored.  I think you mean to make this a raise.